tests/prod_env_tests/algorithm_validation_tests/test_one_way_anova.py

```diff
@@ -0,0 +1,70 @@
+import json
+from pathlib import Path
+
+import numpy as np
+import pytest
+import requests
+
+expected_file = Path(__file__).parent / "expected" / "one_way_anova_expected.json"
+
+
+def anova_one_way_request(input):
+    url = "http://127.0.0.1:5000/algorithms" + "/one_way_anova"
```

Comment:
minor:
You can get this url from here: https://github.com/madgik/MIP-Engine/blob/master/tests/prod_env_tests/__init__.py